component {
  try {
  } catch (org.example.whateverError) {
  }
}